frontend/src/components/ShootWorkers/GWorkerInputGeneric.vue

```diff
@@ -286,64 +261,70 @@ export default {
   },
   data () {
     return {
-      validationErrors,
       immutableZones: undefined,
       volumeSize: undefined,
     }
   },
   validations () {
-    return this.validators
+    const rules = { worker: {} }
+
+    const nameRules = {
+      required,
+      maxLength: maxLength(15),
+      lowerCaseAlphaNumHyphen,
+      noStartEndHyphen,
+      uniqueWorkerName,
+    }
+    rules.worker.name = withFieldName(() => `${this.workerGroupName} Name`, nameRules)
+
+    rules.worker.minimum = withFieldName(() => `${this.workerGroupName} Autoscaler Min.`, {
+      minValue: minValue(0),
+    })
+
+    const maximumRules = {
+      minValue: minValue(0),
+      systemComponents: withMessage('Value must be greater or equal to the number of zones configured for this pool',
+        (value) => {
```

Comment:
I think we should install the `@stylistic/eslint-plugin-js` plugin and enforce a consistent code style via eslint.

Reply:
yes, we should do this in a separate PR